plugins/README.md

```diff
@@ -20,3 +20,27 @@ Go to File -> Settings -> Plugins -> Install from disk and select
 reval.jar. You will need to restart the IDE to see changes. By default
 the keymap shortcut is Ctrl + Alt + S, but you can change it on
 File -> Settings -> Keymap -> Main menu -> Code -> Update Blaze Code
+
+
+### Atom
+Go to Edit -> [Init Script..., Keymaps...]
+In init.coffee paste following (with notion that indentation is important for script):
+```
+http = require('http')
```

Comment:
atom is unique, this file would have nowhere to go unless an actual package, and if using key based patching it's even misleading.